src/Esfa.Vacancy.Api.Types/CreateApprenticeshipParameters.cs

```diff
@@ -34,9 +34,20 @@ public class CreateApprenticeshipParameters
         /// </summary>
         public string WorkingWeek { get; set; }
 
+
         /// <summary>
         /// The number of hours in a typical working week.
         /// </summary>
         public double HoursPerWeek { get; set; }
+        
+        /// <summary>
+        /// The location type used for the vacancy
+        /// </summary>
+        public LocationType LocationType { get; set; }
```

Comment:
These enum inputs for the api request, we should be able to pass in either the numerical representation as well as string. Is this something you could test and verify. Mark said the StrictEnumConverter thing should do it. But I am not sure. If so we should update the acceptance criteria.

Reply:
It works either way, we can update the ACs